### ISSUE TITLE
Mob stacks randomly despawn after 4 of 5 hits

### ISSUE THREAD

**Issue by ArmonicPvP** [BG-Software-LLC/WildStacker#657]
### Minecraft's Version

1.19.2

### Plugin's Version

b379

### Describe the bug

Stacked mobs randomly despawn after 4 or 5 hits. Death behavior seems different now also, as if when they die they make me wait almost a second to kill the next mob and after a few seconds they just completely disappear

### To Reproduce

Kill stacked mobs 4 times

**Comment by OmerBenGera** [comment]
Try replicating the issue on a server with only WildStacker installed.
If it still occurs, take a video of the issue, upload it to an online streaming service (yt for example) and send a link to it here

**Comment by ArmonicPvP** [comment]
> Try replicating the issue on a server with only WildStacker installed. If it still occurs, take a video of the issue, upload it to an online streaming service (yt for example) and send a link to it here

I did a version comparison because it works on 368 but not on 379 or 380

https://youtu.be/YoK4Hk3ZnoE

**Comment by OmerBenGera** [comment]
@ArmonicPvP Interesting.
Please upload your config to pastebin and send it here, I'll try to replicate the issue myself.

**Comment by ArmonicPvP** [comment]
https://pastebin.com/WY6Q6j37

Sorry, I thought I uploaded this already.

**Comment by OmerBenGera** [comment]
You have both blacklist and whitelist for entities, makes no sense.
Also, you have a stack interval of 10. Let's do the following:
1) Empty the whitelist of entities (Set it to `whitelist: []`)
2) Set the `stack-interval` of entities to 0.

Make sure you do it when the server is off, then enable the server again.
Let me know if it fixes the issue.
I believe that the stack-interval caused the entities to get stacked with another.
The whitelist and blacklist together can lead to other issues, so I don't want you to have it neither.

**Comment by ArmonicPvP** [comment]
If I change stack-interval to 0 then my spawner mobs do not properly stack and just create a bunch of additional stacks. 
The whitelist and blacklist is there to whitelist or allow only spawner mobs to stack, but blacklist specific types of spawner mobs from stacking. Is there a way to make the Whitelist whitelist only certain spawner mobs like - SPAWNER:BLAZE for example?

**Comment by OmerBenGera** [comment]
> If I change stack-interval to 0 then my spawner mobs do not properly stack and just create a bunch of additional stacks. The whitelist and blacklist is there to whitelist or allow only spawner mobs to stack, but blacklist specific types of spawner mobs from stacking. Is there a way to make the Whitelist whitelist only certain spawner mobs like - SPAWNER:BLAZE for example?

Yes. You can use in the whitelist "BLAZE:SPAWNER" to whitelist only blazes from spawners to spawn.

Disable the stack interval, check if the issue still occurs or not.

**Comment by ArmonicPvP** [comment]
Didn't touch stack-interval but changed whitelist information and it doesn't happen anymore now it seems.

![image](https://user-images.githubusercontent.com/24983808/200152936-46f2ea42-76b1-49bd-9a83-e2629fda0775.png)
This is what happens when I change it to stack-interval: 0. None of the blazes attempt to stack with each other even though they are spawning within 25 blocks according to my ```
merge-radius:
    BLAZE:SPAWNER: 25
    CHICKEN:SPAWNER: 5
    PIG:SPAWNER: 5
```

I can open a new ticket for this issue. They seem to all be fine if it's within 5 blocks or so

**Comment by OmerBenGera** [comment]
Check if they can stack together with `/ws simulate`

**Comment by ArmonicPvP** [comment]
Okay, after restarting the server my blazes are messed up again and the console is outputting this again and I am unable to fast-kill mobs:

```
[06.11 22:04:00] [Server] [INFO] Named entity EntityBlaze['Blaze (1622)'/4725369, uuid='9011c8ea-ff40-43af-b8c8-87f1428230ed', l='ServerLevel[world]', x=-18263.30, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=3214, v=true] died: Blaze (1622) was slain by DesmondCheese
[06.11 22:04:02] [Server] [INFO] Named entity EntityBlaze['Blaze (1621)'/4737183, uuid='50b2addf-ce1f-49ec-a1e4-5814263ad258', l='ServerLevel[world]', x=-18263.30, y=53.08, z=-7624.70, cpos=[-1142, -477], tl=36, v=true] died: Blaze (1621) was slain by DesmondCheese
[06.11 22:04:07] [Server] [INFO] [Power|Louisville] [Mod] ACE1200: i have see this in my profits, much appreciated
[06.11 22:04:10] [Server] [INFO] Named entity EntityBlaze['Blaze (1615)'/4737295, uuid='6eb0d806-6cc7-4f34-b413-ae0afea732c7', l='ServerLevel[world]', x=-18263.30, y=53.21, z=-7624.70, cpos=[-1142, -477], tl=167, v=true] died: Blaze (1615) was slain by DesmondCheese
[06.11 22:04:12] [Server] [INFO] Named entity EntityBlaze['Blaze (1614)'/4737832, uuid='cb6f23f7-aa4f-429e-8ad4-8eeaf760b096', l='ServerLevel[world]', x=-18263.63, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=42, v=true] died: Blaze (1614) was slain by DesmondCheese
```

/ws simulate returns SUCCESS

**Comment by OmerBenGera** [comment]
> Okay, after restarting the server my blazes are messed up again and the console is outputting this again and I am unable to fast-kill mobs:
> 
> ```
> [06.11 22:04:00] [Server] [INFO] Named entity EntityBlaze['Blaze (1622)'/4725369, uuid='9011c8ea-ff40-43af-b8c8-87f1428230ed', l='ServerLevel[world]', x=-18263.30, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=3214, v=true] died: Blaze (1622) was slain by DesmondCheese
> [06.11 22:04:02] [Server] [INFO] Named entity EntityBlaze['Blaze (1621)'/4737183, uuid='50b2addf-ce1f-49ec-a1e4-5814263ad258', l='ServerLevel[world]', x=-18263.30, y=53.08, z=-7624.70, cpos=[-1142, -477], tl=36, v=true] died: Blaze (1621) was slain by DesmondCheese
> [06.11 22:04:07] [Server] [INFO] [Power|Louisville] [Mod] ACE1200: i have see this in my profits, much appreciated
> [06.11 22:04:10] [Server] [INFO] Named entity EntityBlaze['Blaze (1615)'/4737295, uuid='6eb0d806-6cc7-4f34-b413-ae0afea732c7', l='ServerLevel[world]', x=-18263.30, y=53.21, z=-7624.70, cpos=[-1142, -477], tl=167, v=true] died: Blaze (1615) was slain by DesmondCheese
> [06.11 22:04:12] [Server] [INFO] Named entity EntityBlaze['Blaze (1614)'/4737832, uuid='cb6f23f7-aa4f-429e-8ad4-8eeaf760b096', l='ServerLevel[world]', x=-18263.63, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=42, v=true] died: Blaze (1614) was slain by DesmondCheese
> ```
> 
> /ws simulate returns SUCCESS

Not related to WS, this is not a message from the plugin.

**Comment by OmerBenGera** [comment]
> Okay, after restarting the server my blazes are messed up again and the console is outputting this again and I am unable to fast-kill mobs:
> 
> ```
> [06.11 22:04:00] [Server] [INFO] Named entity EntityBlaze['Blaze (1622)'/4725369, uuid='9011c8ea-ff40-43af-b8c8-87f1428230ed', l='ServerLevel[world]', x=-18263.30, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=3214, v=true] died: Blaze (1622) was slain by DesmondCheese
> [06.11 22:04:02] [Server] [INFO] Named entity EntityBlaze['Blaze (1621)'/4737183, uuid='50b2addf-ce1f-49ec-a1e4-5814263ad258', l='ServerLevel[world]', x=-18263.30, y=53.08, z=-7624.70, cpos=[-1142, -477], tl=36, v=true] died: Blaze (1621) was slain by DesmondCheese
> [06.11 22:04:07] [Server] [INFO] [Power|Louisville] [Mod] ACE1200: i have see this in my profits, much appreciated
> [06.11 22:04:10] [Server] [INFO] Named entity EntityBlaze['Blaze (1615)'/4737295, uuid='6eb0d806-6cc7-4f34-b413-ae0afea732c7', l='ServerLevel[world]', x=-18263.30, y=53.21, z=-7624.70, cpos=[-1142, -477], tl=167, v=true] died: Blaze (1615) was slain by DesmondCheese
> [06.11 22:04:12] [Server] [INFO] Named entity EntityBlaze['Blaze (1614)'/4737832, uuid='cb6f23f7-aa4f-429e-8ad4-8eeaf760b096', l='ServerLevel[world]', x=-18263.63, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=42, v=true] died: Blaze (1614) was slain by DesmondCheese
> ```
> 
> /ws simulate returns SUCCESS

Then they might be too far away when they spawned.
Try enabling linked entities, should help with that.

**Comment by ArmonicPvP** [comment]
I do have linked entities on. It is set to 15 but they stop stacking after the stack is about 5 blocks away every time.

**Comment by ArmonicPvP** [comment]
> > Okay, after restarting the server my blazes are messed up again and the console is outputting this again and I am unable to fast-kill mobs:
> > ```
> > [06.11 22:04:00] [Server] [INFO] Named entity EntityBlaze['Blaze (1622)'/4725369, uuid='9011c8ea-ff40-43af-b8c8-87f1428230ed', l='ServerLevel[world]', x=-18263.30, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=3214, v=true] died: Blaze (1622) was slain by DesmondCheese
> > [06.11 22:04:02] [Server] [INFO] Named entity EntityBlaze['Blaze (1621)'/4737183, uuid='50b2addf-ce1f-49ec-a1e4-5814263ad258', l='ServerLevel[world]', x=-18263.30, y=53.08, z=-7624.70, cpos=[-1142, -477], tl=36, v=true] died: Blaze (1621) was slain by DesmondCheese
> > [06.11 22:04:07] [Server] [INFO] [Power|Louisville] [Mod] ACE1200: i have see this in my profits, much appreciated
> > [06.11 22:04:10] [Server] [INFO] Named entity EntityBlaze['Blaze (1615)'/4737295, uuid='6eb0d806-6cc7-4f34-b413-ae0afea732c7', l='ServerLevel[world]', x=-18263.30, y=53.21, z=-7624.70, cpos=[-1142, -477], tl=167, v=true] died: Blaze (1615) was slain by DesmondCheese
> > [06.11 22:04:12] [Server] [INFO] Named entity EntityBlaze['Blaze (1614)'/4737832, uuid='cb6f23f7-aa4f-429e-8ad4-8eeaf760b096', l='ServerLevel[world]', x=-18263.63, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=42, v=true] died: Blaze (1614) was slain by DesmondCheese
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > /ws simulate returns SUCCESS
> 
> Not related to WS, this is not a message from the plugin.

Yes, but it happens when a named entity is killed. This message seems to pop up every time fast-kill fails which is usually after I kill 3 or 4 of the stack.

**Comment by OmerBenGera** [comment]
> > > Okay, after restarting the server my blazes are messed up again and the console is outputting this again and I am unable to fast-kill mobs:
> > > ```
> > > [06.11 22:04:00] [Server] [INFO] Named entity EntityBlaze['Blaze (1622)'/4725369, uuid='9011c8ea-ff40-43af-b8c8-87f1428230ed', l='ServerLevel[world]', x=-18263.30, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=3214, v=true] died: Blaze (1622) was slain by DesmondCheese
> > > [06.11 22:04:02] [Server] [INFO] Named entity EntityBlaze['Blaze (1621)'/4737183, uuid='50b2addf-ce1f-49ec-a1e4-5814263ad258', l='ServerLevel[world]', x=-18263.30, y=53.08, z=-7624.70, cpos=[-1142, -477], tl=36, v=true] died: Blaze (1621) was slain by DesmondCheese
> > > [06.11 22:04:07] [Server] [INFO] [Power|Louisville] [Mod] ACE1200: i have see this in my profits, much appreciated
> > > [06.11 22:04:10] [Server] [INFO] Named entity EntityBlaze['Blaze (1615)'/4737295, uuid='6eb0d806-6cc7-4f34-b413-ae0afea732c7', l='ServerLevel[world]', x=-18263.30, y=53.21, z=-7624.70, cpos=[-1142, -477], tl=167, v=true] died: Blaze (1615) was slain by DesmondCheese
> > > [06.11 22:04:12] [Server] [INFO] Named entity EntityBlaze['Blaze (1614)'/4737832, uuid='cb6f23f7-aa4f-429e-8ad4-8eeaf760b096', l='ServerLevel[world]', x=-18263.63, y=52.50, z=-7624.70, cpos=[-1142, -477], tl=42, v=true] died: Blaze (1614) was slain by DesmondCheese
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > /ws simulate returns SUCCESS
> > 
> > 
> > Not related to WS, this is not a message from the plugin.
> 
> Yes, but it happens when a named entity is killed. This message seems to pop up every time fast-kill fails which is usually after I kill 3 or 4 of the stack.

Every mob is named, so find out what causes this message to occur.
Try it on a server with only WS installed.

**Comment by ArmonicPvP** [comment]
Issue does not happen on older versions. It only happens whenever the fast-kill is failing and the mobs don't allow me to spam click on them after death

**Comment by ArmonicPvP** [comment]
Any updates on this? Players can't grind

**Comment by OmerBenGera** [comment]
You havent tried it with only WS installed yet.

**Comment by ArmonicPvP** [comment]
Latest dev version seems to have fixed the issue again. Not sure what is going on, but I will test more

**Comment by ArmonicPvP** [comment]
Now after restarting I killed 4 of them and they are invincible and I cannot kill them anymore

**Comment by ArmonicPvP** [comment]
Think I found the problem, it appears to be items from MMOItems/MythicLib and possibly the way they interact with this plugin and the mobs dying. When I use an item from MMOItems like a custom sword it always happens, but never with a regular sword. The mobs will die slowly or turn into a half-dead entity for some reason. I will get dev builds from them to do testing and give updates

**Comment by OmerBenGera** [comment]
@ArmonicPvP Try latest versions, let me know if it still occurs.

**Comment by OmerBenGera** [comment]
Check out latest dev build, I fixed a similar issue that might fixed this one as well.
